script/application.go

```diff
@@ -119,13 +122,59 @@ func (a *Application) RenderProblems() string {
 	return strings.Join(problemStrings, "\n")
 }
 
-func (a *Application) GetData() string {
+func (a *Application) GetData() []byte {
 	data, err := json.MarshalIndent(a, "", "\t")
 	if err != nil {
 		log.Fatalf("Could not marshal Application data: %s", err.Error())
 	}
 
-	return string(data)
+	return data
+}
+
+// Take the application issue number and project name and turn it into
```

Comment:
[idiomatic] Start your method comment with the func name

```suggestion
// FileName takes the application issue number and project name and turns it into
```